src/v2/components/ConnectableContextMenu/index.tsx

```diff
@@ -34,7 +41,8 @@ export const ConnectableContextMenu: React.FC<Props> = ({
 
   return (
     <ContextMenu position="absolute" top={8} right={8} zIndex={1} {...rest}>
-      {channel.can.remove_connections && (
+      {(channel.can.remove_connections ||
+        (connectable.__typename !== 'Channel' && connectable.can.remove)) && (
```

Comment:
One should be able to remove a channel they own that is connected to another channel as well

Reply:
Yeah let's talk about this one...